BD-4874: Clarify Audience Sync to Google is for Google Ads, not Google Ads Manager (#9949)
braze-inc/braze-docs: _docs/_partners/canvas_audience_sync/google_audience_sync.md

- `$google_ad_user_data`
- `$google_ad_personalization`

In any Canvas where you're targeting EEA, UK, and Switzerland end users using a Google Audience Sync to add users to an audience, you need to exclude these users whenever both consent attributes are any value that isn't `true`. This can be achieved by segmenting these users when the consent values are set to `true`. This also ensures that the more accurate analytics of users are synced, since we know Google will reject these users from the audiences. Note that if you're using Google Audience Sync to remove users from an audience, consent attributes are not required.

## Integration

### Step 1: Connect Google account

To get started, go to **Partner Integrations** > **Technology Partners** > **Google Ads** and select **Connect Google Ads**. You'll be prompted with a modal to select the email associated with your Google Ads account and then grant Braze access to your Google Ads account.

> Once you have successfully connected your Google Ads account, you'll be taken back to your Google Ads partner page. You'll then be prompted to select which ad accounts you want to access in the Braze workspace.

Author: ```suggestion
After successfully connecting your Google Ads account, you'll be taken back to your Google Ads partner page. You'll then be prompted to select which ad accounts you want to access in the Braze workspace.
```
